Gulpfile.js

```diff
@@ -211,7 +213,7 @@ gulp.step('travis-saucelabs-qunit', () => {
 gulp.task('test-client-travis', gulp.series('build', 'travis-saucelabs-qunit'));
 
 gulp.step('http-playground-server', () => {
-    return runPlayground();
+    return runPlayground({ needBeautifyScripts });
```

Comment:
What about creating a separate gulp task with beatified script?

Reply:
This flag is usable for all playground tasks . We'll create a lot of tasks if combine playground tasks with the beatify.

Reply:
Ok